zalando-cloud-aws-kms/README.md

```diff
@@ -106,37 +115,36 @@ AWS KMS supports [symmetric and asymmetric keys](https://docs.aws.amazon.com/kms
 
 #### Encryption
 
-Add `keyId` and `encryptionAlgorithm` to the `bootstrap.yaml`:
+Add `key-id` and `encryption-algorithm` to the `bootstrap.yml`:
 
-```
-encrypt.kms:
-  keyId: "9d9fca31-54c5-4df5-ba4f-127dfb9a5031"
-  encryptionAlgorithm: "RSAES_OAEP_SHA_256"  # or "RSAES_OAEP_SHA_1"
+```yaml
+encrypt:
+  kms:
+    key-id: "9d9fca31-54c5-4df5-ba4f-127dfb9a5031"
+    encryption-algorithm: "RSAES_OAEP_SHA_256"  # or "RSAES_OAEP_SHA_1"
 ```
 
 
 #### Decryption
 
-If all cipher values of your application have been encrypted with the
-same KMS key and algorithm, you can configure the `keyId` and `encryptionAlgorithm`
-globally in the `bootstrap.yaml` as shown above. In case you have to decrypt
-ciphers from different keys or different algorithms, you can specify those
-separately for each key using the ["extra options"](#use-extra-options) approach:
-
-e.g. `application.yml`
+If all cipher values of your application have been encrypted with the same KMS key and algorithm, you can configure 
+the `keyId` and `encryptionAlgorithm` globally in the `bootstrap.yml` as shown above. In case you have to decrypt
```

Comment:
The use of `keyId` and `encryptionAlgorithm` here were left as I was not certain how the "extra options" might be defined in the YAML, i.e., should the be kebab-case or camelCase?